sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsActionDetails.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using Azure.AI.TextAnalytics.Models;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// TextAnalyticsActionDetails.
+    /// </summary>
+    public class TextAnalyticsActionDetails
+    {
+        /// <summary>
+        /// TextAnalyticsActionDetails
+        /// </summary>
+        internal TextAnalyticsActionDetails (DateTimeOffset completedOn, IReadOnlyList<TextAnalyticsErrorInternal> errors, bool hasError)
+        {
+            CompletedOn = completedOn;
+            Errors = Transforms.ConvertToErrors(errors);
+            HasError = hasError;
+        }
+
+        /// <summary>
+        /// CompletedOn
+        /// </summary>
+        public DateTimeOffset CompletedOn { get; }
+
+        /// <summary>
+        /// Error
+        /// </summary>
+        public IReadOnlyList<TextAnalyticsError> Errors { get; }
+
+        /// <summary>
+        /// HasError
+        /// </summary>
+        public bool HasError { get; }
```

Comment:
it should be something like https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsResult.cs#L48